plugin/http/Encode.lua

```diff
@@ -0,0 +1,23 @@
+local HttpService = game:GetService("HttpService")
+
+
+local Encode = {}
+
+function Encode.jsonEncode(object)
+	-- Encode json 5 IEEE 754 tokens
+	local body = HttpService:JSONEncode(object)
```

Comment:
I don't want to use regex here. But given the limited number of non-quoted tokens we need to handle it seems superior to alternatives

Reply:
I just realized I can probably refine this to handle all non-quotes string tokens in one pattern with a group